### DISCUSSION TITLE
Wrapper to get jest-openapi compatibility

### DISCUSSION THREAD

**Discussion by SchabaJo** [openapi-ts/openapi-typescript#1271] [Show and tell]
Hi!

I really love `openapi-typescript` and `openapi-fetch` and I really love `jest-openapi`.
However, these projects are not compatible, as `jest-openapi` expects a `superagent`-like response.
Here's my approach to reach compatibility: https://gist.github.com/SchabaJo/b012605f6d5d6f5b0fb99ea4e8de5b10

Have a nice day and keep up the great work!

**Comment by drwpow** [comment]
Thanks for raising this! I will take a look at `jest-openapi` and your wrapper. Though I don’t think we’d change the API to a `superagent`-like response, I think a wrapper would make sense. We could start with it in the documentation, and if it’s helpful enough it could be published as its own npm package (if that’s something you’re interested in publishing, by all means!).

And while investigating, if I run across any type tweaks that would make it easier to consume with `jest-openapi` I’ll keep an eye out
